10_datastructures_lab/01.py

```diff
@@ -1,10 +1,34 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+"""
+The program receives hosts file as input with format: hostName=ipAddress
+and lets the user choose a host and get its ip address.
+If the host is not in the list, a message is shown to the user
+"""
+
+import sys
+
+entries = {}
+
+if (len(sys.argv) != 2):
+    exit("Usage: 01.py <hostsFile>")

```

Comment:
prefer `sys.exit()` over `exit`. There's a detailed discussion and explanation on their differences here:
http://stackoverflow.com/questions/6501121/the-difference-between-exit-and-sys-exit-in-python